packages/drivers/routerlicious-driver/src/documentDeltaConnection.ts

```diff
@@ -46,7 +47,7 @@ export class R11sDocumentDeltaConnection extends DocumentDeltaConnection
             versions: protocolVersions,
         };
 
-        const deltaConnection = new R11sDocumentDeltaConnection(socket, id, logger);
+        const deltaConnection = new R11sDocumentDeltaConnection(socket, id, logger, true);
```

Comment:
nit: I think it would be more readable:
, true /* enableLongPollingDowngrades */).

Reply:
Thanks for the suggestion! I put the value in a `const` with a TODO to expose it to the host via factory policies.